dolphinscheduler-ui/src/js/conf/home/pages/projects/pages/definition/pages/list/_source/timing.vue

```diff
@@ -160,12 +176,14 @@
         processDefinitionId: 0,
         failureStrategy: 'CONTINUE',
         warningTypeList: warningTypeList,
+        availableTimezoneIDList: availableTimezoneIDList,
         warningType: 'NONE',
         notifyGroupList: [],
         warningGroupId: '',
         spinnerLoading: false,
         scheduleTime: '',
         crontab: '0 0 * * * ? *',
+        timezoneId: Intl.DateTimeFormat().resolvedOptions().timeZone,
```

Comment:
Do not use `Intl`, since it is not supported on old browsers such as IE 11. We need to consider the browser compatibility in next release. You can consider `dayjs` or `momentjs`, and `dayjs` has been introduced in https://github.com/apache/dolphinscheduler/blob/dev/dolphinscheduler-ui/src/js/module/filter/filter.js

Reply:
Done, use `moment-timezone` lib to get timezone